cmd/core-command/Dockerfile

```diff
@@ -49,5 +51,13 @@ WORKDIR /
 COPY --from=builder /edgex-go/cmd/core-command/Attribution.txt /
 COPY --from=builder /edgex-go/cmd/core-command/core-command /
 COPY --from=builder /edgex-go/cmd/core-command/res/configuration.toml /res/configuration.toml
-ENTRYPOINT ["/core-command"]
-CMD ["-cp=consul.http://edgex-core-consul:8500", "--registry", "--confdir=/res"]
+
+ENV CONSUL_HOST=edgex-core-consul
+ENV CONSUL_PORT=8500
```

Comment:
Why are these needed? I though the entry point script was going to wait on a port on the new bootstrap service?

Reply:
these two ENVs are used in the `consul_boot_wait_install.sh` shell script so that they are not hard-coded everywhere in the script.

Reply:
Ok, if we use REGISTRY_HOST & REGISTRY_PORT instead in the scripts we can rely on the exist common.env settings. Just need to add REGISTRY_PORT=8500 to it. 

If you still rather have CONSUL_HOST & CONSUL_PORT, lets add then to common.env once instead of to all the Dockerfiles.